main.cpp

```diff
@@ -103,4 +107,9 @@ int main(int argc, char** args) {
         }
     }
     PetscFinalize() >> checkError;
+
+    // output runtime
+    clock_t stop = clock();
+    double elapsed = (double)(stop - start) / CLOCKS_PER_SEC;
+    std::cout << "Runtime: " << elapsed << " sec" << std::endl;
```

Comment:
@maesemen I would prefer if we needed add any required output that is not controlled by the user.  If we wanted to do this I think that we should allow them to specify if/how the output is created.  However, this functionality is already covered in PETSc with more detail.  Have you used  log_view https://petsc.org/main/docs/manualpages/Profiling/PetscLogView.html ?

Reply:
I took it out for now.  I have not used log_view but I will take a look.